app/frontend/stylesheets/pages/_index.scss

```diff
@@ -1,3 +1,4 @@
 @import "partials/calendar";
 @import "partials/membership";
+@import "partials/event-card";
```

Comment:
Please move the event-card import between the calendar and the membership and rename the file `_event_cards.scss`